woocommerce/woocommerce
# Product variation order should be persisted on save

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/woocommerce/issues/36060






### How to test the changes in this Pull Request:



1. Create a product with some variations using the legacy experience and grab its id
2. Got to `/wp-admin/admin.php?page=wc-admin&path=/product/{productId}`
3. Under `Options` tab in `Variations` section you can drag and drop each variation.
4. If you have more than 25 variations you can performe ordering in another page.
5. After clicking the update button all ordering operations across pages should be persisted.

<video src="https://user-images.githubusercontent.com/13334210/208758539-b55b82ae-d2b1-4faf-805a-ddc182c2653e.mov"></video>



### Other information:




### FOR PR REVIEWER ONLY:
